cms/djangoapps/contentstore/views/item.py

```diff
@@ -628,6 +625,10 @@ def _save_xblock(user, xblock, data=None, children_strings=None, metadata=None,
             # IMPORTANT NOTE: if the client passed 'null' (None) for a piece of metadata that means 'remove it'. If
             # the intent is to make it None, use the nullout field
             if metadata is not None:
+                discussion_enabled = metadata.pop('discussion_enabled', None)
+                if discussion_enabled is not None:
+                    set_discussion_toggle(discussion_enabled, xblock, user)
+
```

Comment:
I think these changes should probably be moved to the other PR. Or we can perhaps merge them both into a singe multi-author PR.

Reply:
+1 to this, I'd love to see this in #24380. I can extract this to a separate commit for cherry-picking, if needed.

Reply:
I can start looking at the new pieces in this PR, but maybe we can rebase this over the other PR and then update the target/base branch of this PR to be a PR _into_ the other PR branch?

Reply:
@stvstnfrd, the problem with this approach is that both PRs are opened from our fork, so AFAIK it's not possible to change the target to this one - we would need to have two different PRs (for our fork and your upstream repository; side effect would be having the comments on only one PR and not having your CI pipelines running). We could also push `open-craft:ahmed/bb-2555-enable-discussion-toggle` to edx remote, but then we would need to keep both branches in sync.